scripts/asm-installer/install_asm

```diff
@@ -728,6 +733,47 @@ validate_istio_version() {
   fi
 }
 
+validate_asm_version() {
+  info "Checking existing ASM version(s)..."
+  local OUTPUT
+  OUTPUT="$(retry 3 ./istio-${RELEASE}/bin/istioctl version -o json)"
+  local FOUND_INVALID_VERSION
```

Comment:
I think the style consistent with the rest of the script is `local X; X=something;` for shorter lines.

Reply:
Currently, it's mixed style. I was following the existing **validate_istio_version** as a reference but I will updated both to match this.

Reply:
Fair: I think the rule of thumb is that if you can do it and stay within 80 char width then do it,  otherwise split them. If you think it's more readable like this then feel free to keep it.